com.vscodium.codium-insiders.yaml

```diff
@@ -0,0 +1,143 @@
+id: com.vscodium.codium-insiders
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+command: codium-insiders
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --device=all
+  - --filesystem=host
+  - --persist=.vscode-oss
+  - --allow=devel
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Flatpak
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --filesystem=xdg-run/gnupg:ro
+  - --filesystem=xdg-config/kdeglobals:ro
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.AppMenu.Registrar.*
+  - --system-talk-name=org.freedesktop.login1
+separate-locales: false
+add-extensions:
+  com.visualstudio.code.tool:
+    directory: tools
+    subdirectories: true
+    version: '23.08'
+    add-ld-path: lib
+    no-autodownload: true
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - '*.la'
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - shared-modules/libusb/libusb.json
+  - name: codium-insiders
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 codium-insiders.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - |
+        for size in 16 32 48 64 128 256 512; do
+          rsvg-convert --width ${size} --keep-aspect-ratio --format png --output codium-insiders_${size}.png codium-insiders.svg || exit 1
+          install -Dm644 codium-insiders_${size}.png /app/share/icons/hicolor/${size}x${size}/apps/$FLATPAK_ID.png || exit 1
+        done
```

Comment:
you don't need to do that. Just install the SVG.

Reply:
Is really a problem to have the complete set?

Reply:
it is unecessary. it complicates the build for no reason. the SVG is enough.

Reply:
> it is unecessary. it complicates the build for no reason. the SVG is enough.

The freedesktop specs say that [only SVG is optional](https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html#definitions), and at least a 48px PNG or XPM is required for scalable.

But if this is what you request, the PNGs are already removed

Reply:
appstream compose will generate the png from svg as required.

Reply:
> appstream compose will generate the png from svg as required.

Thanks for this!